test/iron-menu-behavior.html

```diff
@@ -21,7 +21,7 @@
 
     <script src="../../web-component-tester/browser.js"></script>
     <script src="../../test-fixture/test-fixture-mocha.js"></script>
```

Comment:
This is orthogonal to the change, but can you set the `web-component-tester` version to `v3.4.0` and remove `test-fixture-mocha.js` and `test-fixture.html` from the test suite?